tests/regression_functional_test.go

```diff
@@ -0,0 +1,122 @@
+package tests_test
+
+import (
+	"flag"
+	"github.com/google/goexpect"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"kubevirt.io/kubevirt-ansible/tests"
+	"time"
+)
+
+//// The list of supported Operating Systems
+//// name : path_to_yml
+var osDict = map[string]string{
+	"testvm-fedora" : "manifests/vm-template-fedora.yml",
+}
+
+const (
+	temporaryJson = "/tmp/tmp-vm.yml"
+	ipQuery = "-o=jsonpath='{.status.interfaces[0].ipAddress}'"
+	phaseQuery = "-o=jsonpath='{.status.phase}'"
+)
+
+
+var _ = Describe("Regression and Functional tests of VMs and VMIs", func() {
+
+	flag.Parse()
```

Comment:
Why are you parsing command line flags? I do not see it used anywhere.

Reply:
the test is run with parameter from CLI [1], so we need to parse them
[1]
${TESTS_OUT_DIR}/tests.test -kubeconfig=$kubeconfig -tag=$tag -prefix=$prefix -oc-path=${OC_PATH} -kubectl-path=${KUBECTL_PATH} -virtctl-path=${VIRTCTL_PATH} ${FUNC_TEST_ARGS}